src/components/organisms/leporello-graphic-real-item/leporello-graphic-real-item.jsx

```diff
@@ -54,85 +56,103 @@ export default ({
         </div>
 
         <div className="leporello-graphic-real-item__info">
-          <GroupedDefinitionList>
-            <GroupedDefinitionList.Entry
-              term="CDA ID"
-              definition={inventoryNumber}
-            />
-            <GroupedDefinitionList.Entry
-              term={ t('Dating') }
-              definition={dating.dated}
-            />
-          </GroupedDefinitionList>
+
+          <h2 className="leporello-graphic-real-item__title">{title}, {dating.dated}</h2>
 
           {/* Physikalische Eigenschaften */}
           <GroupedDefinitionList>
             <GroupedDefinitionList.Entry
-              term={ t('Classification') }
+              term={t('Classification')}
               definition={classification}
             />
             <GroupedDefinitionList.Entry
-              term={ t('Medium') }
+              term={t('Medium')}
               definition={medium}
             />
             <GroupedDefinitionList.Entry
-              term={ t('Measurments') }
+              term={t('Dimensions')}
               definition={dimensions}
             />
           </GroupedDefinitionList>
 
           {/* Besitz, Eigentümer und so */}
           <GroupedDefinitionList>
             <GroupedDefinitionList.Entry
-              term={ t('Owner') }
+              term={t('Owner')}
               definition={owner}
             />
             <GroupedDefinitionList.Entry
-              term={ t('Repository') }
+              term={t('Repository')}
               definition={repository}
             />
             <GroupedDefinitionList.Entry
-              term={ t('Location') }
+              term={t('Location')}
               definition={location}
             />
           </GroupedDefinitionList>
 
           {/* Inschriften, Texte und so */}
-         { (signature || inscription || markings || description || provenance)
+          {(signature || inscription || markings || description)
```

Comment:
Wenn ich es richtig sehe, hat sich `description` nachfolgend zu `provenance` und `exhibitionHistory` hinzugesellt. Kann dann hier bestimmt neben `provenance` ebenfalls aus der OR-Bedingungskette entfernt werden.

```suggestion
          {(signature || inscription || markings)
```